### PR TITLE
uses templates instead of inline sql

### PR DESCRIPTION
**Pull request by mmm**
externalizes views as well as schema so this will hopefully be easier to read and maintain

### CODE REVIEW

**Comment by Crash-GHaun** [comment]
@mmm Please review the changes I made. I ran it and everything seems to be good but I wanted to triple check

File: recommendations-dashboard/terraform/modules/recommendations-dashboard/bigquery_schemas/recommendations_export_schema.sql [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  {"description":"Ancestry for the recommendation entity\\n","fields":[
+    {"description":"Organization to which the recommendation project\\n","mode":"NULLABLE","name":"organization_id","type":"STRING"},
+    {"description":"Up to 5 levels of parent folders for the recommendation project\\n","mode":"REPEATED","name":"folder_ids","type":"STRING"}
+  ],"mode":"NULLABLE","name":"ancestors","type":"RECORD"},
+  {"description":"Insights associated with this recommendation. A project insight is represented as\\nprojects/[PROJECT_NUMBER]/locations/[LOCATION]/insightTypes/[INSIGHT_TYPE_ID]/insights/[insight_id]\\n","mode":"REPEATED","name":"associated_insights","type":"STRING"},
+  {"description":"Additional details about the recommendation in JSON format\\n","mode":"NULLABLE","name":"recommendation_details","type":"STRING"},
+  {"description":"Priority of the recommendation:\\n  PRIORITY_UNSPECIFIED:\\n    Default unspecified priority. Don't use directly.\\n  P4:\\n    Lowest priority.\\n  P3:\\n    Second lowest priority.\\n  P2:\\n    Second highest priority.\\n  P1:\\n    Highest priority.\\n","mode":"NULLABLE","name":"priority","type":"STRING"}
+]
+
```
Comment (mmm) [inline recommendations-dashboard/terraform/modules/recommendations-dashboard/bigquery_schemas/recommendations_export_schema.sql:34]: yeah, the formatting for the `fields` here is just tough... I'm not sure what to do with it

File: recommendations-dashboard/terraform/modules/recommendations-dashboard/bigquery_schemas/insights_export_schema.sql [hunk 2]
```diff
@@ -15,6 +15,6 @@
...
+  {"description":"Additional details about the insight in JSON format\\nschema:\\n  fields:\\n  - name: content\\n    type: STRING\\n    description: |\\n      A struct of custom fields to explain the insight.\\n      Example: \"grantedPermissionsCount\": \"1000\"\\n  - name: observation_period\\n    type: TIMESTAMP\\n    description: |\\n      Observation period that led to the insight. The source data used to\\n      generate the insight ends at last_refresh_time and begins at\\n      (last_refresh_time - observation_period).\\n- name: state_metadata\\n  type: STRING\\n  description: |\\n    A map of metadata for the state, provided by user or automations systems.\\n","mode":"NULLABLE","name":"insight_details","type":"STRING"},
```
Comment (mmm) [inline recommendations-dashboard/terraform/modules/recommendations-dashboard/bigquery_schemas/insights_export_schema.sql:18]: good catch!

**Review by mmm — COMMENTED** [review]
+1 lgtm
(won't let me "approve" it b/c I was the PR's author)

File: recommendations-dashboard/terraform/modules/recommendations-dashboard/bigquery_schemas/recommendations_export_schema.sql (hunk 1 above)
Comment (Crash-GHaun) [reply recommendations-dashboard/terraform/modules/recommendations-dashboard/bigquery_schemas/recommendations_export_schema.sql:34]: Better in it's own file then inline xD

**Pull request merged**